src/applications/accredited-representative-portal/sass/SubmissionsPage.scss

```diff
@@ -0,0 +1,36 @@
+@import "~@department-of-veterans-affairs/css-library/dist/tokens/scss/variables";
```

Comment:
I don't think you need to import the sass variables if you're only using the custom properties in this stylesheet (those are already loaded globally). ie. `--vads-color-success-dark`